packages/ember-glimmer/lib/environment.js

```diff
@@ -31,6 +31,10 @@ import { default as unbound } from './helpers/unbound';
 import { default as classHelper } from './helpers/-class';
 import { OWNER } from 'container/owner';
 
+const keywords = {
```

Comment:
keywords was a specific thing in htmlbars that doesn't existing in glimmer 2.  This is introducing a new concept that non of the other glimmer 2 builtins have used.  I would say builtin component is more what you are describing here.  /cc @wycats